main.go

```diff
@@ -243,7 +252,7 @@ func (svr *bridge) handleCall(w http.ResponseWriter, r *http.Request) {
 
 			metrics["alerts_received"]++
 			if *svr.debug {
-				log.Printf("  Alert %d", idx)
+				log.Printf("Alert %d", idx)
```

Comment:
I would prefer these spaces to be retained since they visually indent the log output so a human reader can more easily see logic branches/steps.

Reply:
Sure no problem. I added those back in.